.env

```diff
@@ -0,0 +1,20 @@
+# Sets the location of the flask application object: https://flask.palletsprojects.com/en/2.3.x/cli/#application-discovery
+# Needed because of the presence of the "todo_app" folder
+FLASK_APP=todo_app/app
+
+# Turn on debug mode (which enables reloading on code changes and the interactive debugger: https://flask.palletsprojects.com/en/2.3.x/config/#DEBUG)
+FLASK_DEBUG=true
+
+# Required to use Flask's session feature (for storing todos in a browser cookie in the first exercise): https://flask.palletsprojects.com/en/2.3.x/config/#SECRET_KEY
+SECRET_KEY=secret-key
+
+#Add your TRELLO API credentials as shown below
+
+TRELLO_API_KEY=ff02dc85cb92b92ec769718e2a6befa6
+TRELLO_API_TOKEN=ATTA03556a259f84874adabae6c9b4d03ce72c84b0917dc56819f7a9a63ebda693536CD66738
```

Comment:
Discussed over Zoom how to deal with this secret leak - the file has now been removed (`git rm --cached .env` for posterity) and this token has been revoked so it can't be used